osf/migrations/0002_auto_20220726_1732.py

```diff
@@ -0,0 +1,25 @@
+# Generated by Django 3.2 on 2022-07-26 17:32
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
```

Comment:
This is necessary due to our addon structure, regions are dependent on the Django osf app, but it's in the osfstorage addon app.